### PR TITLE
Add package for bootstrap-notify

### PR DESCRIPTION
**Pull request by atroche**
New package:

**Extern:** Hand made from library JS

### CODE REVIEW

File: bootstrap-notify/build.boot [hunk 1]
```diff
@@ -0,0 +1,32 @@
+(set-env!
+  :resource-paths #{"resources"}
+  :dependencies '[[cljsjs/boot-cljsjs "0.5.2" :scope "test"]])
```
Comment (Deraen) [inline bootstrap-notify/build.boot]: Based on library bower.json, this library depends on jquery and bootstrap: https://github.com/mouse0270/bootstrap-notify/blob/master/bower.json
This package should thus depend on cljsjs/jquery and cljsjs/bootstrap

File: bootstrap-notify/build.boot [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+             :url (str "https://github.com/mouse0270/bootstrap-notify/archive/" +lib-version+ ".zip")
+             :checksum "93623E5DB0EB7A142A9A21BF7284E3EC"
+             :unzip true)
+           (sift :move {#"^bootstrap-notify-.*/dist/bootstrap-notify.js"
+                        "cljsjs/bootstrap-notify/development/bootstrap-notify.inc.js"
+
+                        #"^bootstrap-notify-.*/dist/bootstrap-notify.min.js"
+                        "cljsjs/bootstrap-notify/production/bootstrap-notify.min.inc.js"})
+           (sift :include #{#"^cljsjs"})
+           (deps-cljs :name "cljsjs.bootstrap-notify")
```
Comment (Deraen) [inline bootstrap-notify/build.boot]: Provided namespace should also depend on `cljsjs.jquery` and `cljsjs.bootstrap`

File: bootstrap-notify/resources/cljsjs/boostrap-notify/common/bootstrap-notify.ext.js [hunk 3]
```diff
@@ -0,0 +1,2839 @@
+/**********************************************************************
+ * Extern for jQuery
+ * Generated by http://jmmk.github.io/javascript-externs-generator
+ **********************************************************************/
+var jQuery = {
```
Comment (Deraen) [inline bootstrap-notify/resources/cljsjs/boostrap-notify/common/bootstrap-notify.ext.js]: The extern file must not redefine vars declared in other extern files, like in jQuery extern. This file should only contain properties added by the library, e.g. `jQuery.notify`, `jQuery.notifyDefaults`, `jQuery.notifyClose` etc.

**Comment by atroche** [comment]
@Deraen thanks for the help! how's that?

**Pull request merged**